src/utils/markdown/rehype-expand-details-summary.ts

```diff
@@ -0,0 +1,43 @@
+import { visit } from "unist-util-visit";
+import { Element, Root } from "hast";
+
+const unicodeElement: Element = {
+	type: "element",
+	tagName: "span",
+	properties: {
+		style: "margin-right: 3px;",
```

Comment:
We have to add some margin because `chevron` and `title` are too close to each other.